doc/Getting-Started-Core.html

```diff
@@ -37,18 +40,37 @@ <h3 id="DocCreateProject">Create a Project</h3>
 
 <h3 id="DocConfigureProject">Configure The Project</h3>
 
+
+<div class="alert alert-warning">
+    <strong>Important Notice:</strong><br>
+    <p>Installing client side npm dependencies using <strong>yarn</strong> before opening the solution will increase project opening &amp; building time dramatically.</p>
```

Comment:
Should this be `decrease`?

Reply:
Thanks @acjh , you are great 😄 . Will fix it with next commit.